brainreg_segment/segmentation_panels/regions.py

```diff
@@ -144,6 +156,28 @@ def add_region(self):
             self.num_colors,
         )
 
+    def add_region_from_existing_layer(self, confirmation=True):
+        print("Adding region from existing layer\n")
+        selected_layer = self.parent.viewer.layers.selection.active
+        if isinstance(selected_layer, Labels):
+            add_region_from_existing_layer(
+                selected_layer, self.parent.label_layers
+            )
+            if confirmation:
+                display_info(
+                    self.parent,
+                    "Layer added",
+                    f"Added layer: {str(selected_layer)}.",
+                )
+        else:
+            if confirmation:
+                display_info(
```

Comment:
Are these confirmation boxes meant to be turned on by default? If I try and add a new region from a points layer, I don't get an info box popping up saying that the layer isn't a labels layer.

Reply:
Fixed in edbf298c76d09d8e987c04dfbe143489008da0e6